feedset_test.go

```diff
@@ -1,22 +1,31 @@
 // SPDX-License-Identifier: MIT
 
-package graph
+package ssb
 
 import (
 	"testing"
 
 	"github.com/stretchr/testify/require"
-	"go.cryptoscope.co/ssb"
 )
 
+func TestFeedSetEmpty(t *testing.T) {
+	r := require.New(t)
+
+	fs := NewFeedSet(0)
+
+	newkey, err := NewKeyPair(nil)
+	r.NoError(err)
+	r.False(fs.Has(newkey.Id))
```

Comment:
why not require.False(..) and require.NoError(..)?  I have not seen require.New(..) before

Reply:
Mostly because i need to type the full package name plus `t,` :upside_down_face: